wasm/instructions.cpp

```diff
@@ -171,4 +174,22 @@ std::optional<std::vector<Instruction>> parse(std::istream &is) {
     }
 }
 
+std::optional<std::string> to_string(Instruction const &inst) {
```

Comment:
How about something like this, where `to_string()` is called recursively to serialize the whole sub-tree? So if you pass in a `Block`, it gives you the whole block, but passing in a `I32Add` gives you just the one `(i32.add)`.

Reply:
Yeah, that sounds great!